.gitignore

```
@@ -0,0 +1,18 @@
+.DS_Store
+Thumbs.db
+node_modules
+*.log
+.*.log
+*.env*
+*.lock*
```

We should allow `deno.lock` files.

```suggestion
*.lock*
!deno.lock
```